### DISCUSSION TITLE
NEXT JS 13 'use client' gives unexpected error

### DISCUSSION THREAD

**Discussion by Inst1nct007** [vercel/next.js#48650] [Help]
### Summary

I'm using the experimental app directory with TypeScript support.

This is a Server component.

```
export default function SimpleComponent() {
    return (
        <>
            <h1>Hello</h1>
        </>
    )
}
```
Now I want to introduce state here. So logically I would have to turn this into a Client component by using the 'use client' directive. Right?

OK. So I turned it into a simple client component with state.

```
"use client";

import { useState } from "react";

export default function SimpleComponent() {
  const [word, setWord] = useState("Simple Client Component");

  function handleClick() {
    setWord("Hello");
  }

  return (
    <>
      <h1 onClick={handleClick}>{word}</h1>
    </>
  );
}
```
When I do this, I get this error in my terminal.

`SyntaxError: Unexpected token u in JSON at position 0 at JSON.parse ()`

And the webpage at `localhost:3000` turns white.

I also posted this issue on Stack Overflow - https://stackoverflow.com/questions/76054383/next-js-13-use-client-gives-unexpected-error and a person was kind enough to help me. He told me to change my next version in `package.json`

**from**
```
"dependencies": {
    "next": "13.3.0"
  }
```

**to**
```
"dependencies": {
    "next": "13.1.1"
  }
```

This solved my issue thanks to him. But I want to know if this is an issue with the current NEXT JS version? or did I do something wrong?

**Comment by danimadmolil** [comment]
Hi. converting all the function decleratoins to arrow functions in pages, layouts, components should fix the problem.i have the same problem and it fixed.

**Reply by Inst1nct007** [reply]
thank you so much for the help.
